tests/pair_potential_calculator.py

```diff
@@ -0,0 +1,71 @@
+#! /usr/bin/env pytho
+
+# ======================================================================
+# matscipy - Python materials science tools
+# https://github.com/libAtoms/matscipy
+#
+# Copyright (2014) James Kermode, King's College London
+#                  Lars Pastewka, Karlsruhe Institute of Technology
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+# ======================================================================
+
+from __future__ import print_function
+
+import random
+import unittest
+import sys
+
+import numpy as np
+from numpy.linalg import norm
+
+import ase.io as io
+from ase.constraints import StrainFilter, UnitCellFilter
+from ase.lattice.compounds import B1, B2, L1_0, L1_2
+from ase.lattice.cubic import FaceCenteredCubic
+from ase.lattice.hexagonal import HexagonalClosedPacked
+from ase.optimize import FIRE
+from ase.units import GPa
+
+import matscipytest
+from matscipy.calculators.pair_potential import PairPotential, LennardJonesCut, LennardJonesQuadratic, get_dynamical_matrix
```

Comment:
Where does `get_dynamical_matrix` come from? I don't see it in this commit. Why is it a function and not a method of the `PairPotential`?